meross_iot/cloud/devices/subdevices/sensors.py

```diff
@@ -0,0 +1,54 @@
+from enum import Enum
+from meross_iot.cloud.abilities import HUB_MS100_ALL, HUB_MS100_TEMPHUM, HUB_ONLINE
+from meross_iot.cloud.devices.subdevices.generic import GenericSubDevice
+from meross_iot.logger import SENSOR_LOGGER as l
+from meross_iot.meross_event import DeviceSwitchStatusEvent, SensorTemperatureChange, DeviceOnlineStatusEvent
+
+
+class SensorSubDevice(GenericSubDevice):
+
+    def __init__(self, cloud_client, subdevice_id, parent_hub, **kwords):
+        super().__init__(cloud_client, subdevice_id, parent_hub, **kwords)
+
```

Comment:
Like commented elsewhere, I'd override here the _sync_status() method providing specific implementation